.github/ISSUE_TEMPLATE/challenge.md

```diff
@@ -0,0 +1,63 @@
+---
+name: Challenge
+about: Add a new challenge.
+title: ''
+labels: ['triage', 'challenge']
+assignees: ''
+---
+<!--
+Give the issue a succinct, attractive title.
+
+Example:
+Greet the user with their preferred name
+-->
+
+## Pitch
+<!--
+Add in concrete terms what the problem is.
+
+Example:
+- Users like to be addressed by their preferred name within apps.
+  - This is a name that they are known as, or use for themselves.
+- In some cultures, this is the first name.
+  - It could also be a user-chosen name or nickname.
+- Other cultures do not have the concept of a first name, so another strategy is needed.
+-->
+
+## Desired solution
+<!--
+Add insights in how the solution should/can look like.
+
+Example:
+- Encode the rules for arriving at the preferred name in a declarative way.
+  - For instance, using reasoning rules.
+- Expose the functionality via JavaScript
+  - For instance, through `getUserPreferredName` or `user.preferredName`
+-->
+
+## Acceptance criteria
+<!--
+Describe the criteria for a demo that needs to be built such that solution can be accepted.
+
+Example:
+- I want to see an example app with at the top right a user bar.
+- This user bar shows a profile picture, and "Hi, Alex".
+  - Here, "Alex" is the logged-in user's preferred name.
+- This demo should work for people from different backgrounds.
+  - people without a first name
+  - people with a nickname
+  - people with a chosen name
+-->
+
+## Pointers
+<!--
+Add pointers to existing solutions, approaches, ideas, and so on.
+
+Example:
+- https://www.w3.org/International/questions/qa-personal-names
+-->
+
+## Scenarios
+<!--
+Add to which scenarios this challenge contributes.
```

Comment:
@pheyvaer I did not get this one; is this SolidLab-specific? Is there a link that we can point to so people can pick?

I could not write an `Example:` section for this one because I didn't know. Can you add this one, or shall we remove that section for now?

Reply:
Considering the flow we discussed: use case --> scenario --> challenge, people should then mention to which scenario the challenge belongs. We can set it optional of course if people can submit challenges that do not belong to a scenario/use case. But do we want that?

Reply:
Ah it's use case; right. I will indeed think how to incorporate this.